internal/pkg/cli/svc_init.go

```diff
@@ -311,22 +313,31 @@ func (o *initSvcOpts) askDockerfile() error {
 		return nil
 	}
 
-	// TODO https://github.com/aws/copilot-cli/issues/206
 	dockerfiles, err := listDockerfiles(o.fs, ".")
-	if err != nil {
+	// If no Dockerfiles are found in current directory or subdirectory one level down, prompt user.
+	var sel string
+	if err != nil && strings.HasPrefix(err.Error(), "no Dockerfiles found") {
```

Comment:
Can we extract the "no Dockerfiles found..." error message into its own type and use the `errors.As()` function to do this check?

Reply:
Yup! Thanks!